File: HeartBeatSystemDemo/HeartBeatSystemDemo/src/Adrenaline.java
public class Adrenaline implements Drags{
    public static final DragType type = DragType.ADRENALINE;
    private int duration;
    public Adrenaline(int duration){
        this.duration = duration;
    }
    @Override
    public int getDuration() {
        return duration;
    }

    @Override
    public DragType getType() {
        return type;
    }
}

File: UnityProjet/scripts/HeartBeatSystem/models/Adrenaline.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Adrenaline : IDrags
{
    public const DragTypes Type = DragTypes.ADRENALINE;
    private readonly int Duration;
    public Adrenaline(int duration)
    {
        this.Duration = duration;
    }
    public int GetDuration()
    {
        return Duration;
    }

    DragTypes IDrags.GetType()
    {
        return Type;
    }

}
